spack/packages/seissol-env/package.py

```diff
@@ -102,5 +102,7 @@ def setup_run_environment(self, env):
         env.prepend_path('PYTHONPATH', ":".join(filter(None, pythonpath)))
         
         # add pspamm while loading seissol-env
-        env.prepend_path('PYTHONPATH', self.spec.dependencies_dict()['pspamm'].spec.prefix.pspamm)
-        env.prepend_path('PATH', self.spec.dependencies_dict()['pspamm'].spec.prefix.pspamm)
+        pspamm = self.spec.dependencies(name='py-pspamm')[0]
```

Comment:
You can also write 
```
pspamm = self.spec['py-pspamm']
```
https://spack.readthedocs.io/en/latest/packaging_guide.html#accessing-dependencies